run_robot_test.sh

```diff
@@ -16,9 +16,17 @@ JENKINS_BUILD_NUMBER=${BUILD_NUMBER:-666}
 
 # -------- functions -------------
 
+override_url_variables() {
+    # Overrides Openshift URL variables with Current Openshift URL values
+    echo "Retrieving Openshift Console & API Server URL Values"
+    console_url=$(oc whoami --show-console)
+    api_server=$(oc whoami --show-server)
```

Comment:
@mrrajan for _oc whoami_ to work, infact for any oc command to work, don't we first need to be logged in to the cluster with OC?
![image](https://user-images.githubusercontent.com/100675023/186597675-9b0d293f-520b-4583-99c6-0b54dc1cadc1.png)

Reply:
@harshkumarRH This block would be executed with the argument "-x" and the intention is to use this argument only when we run this image on Openshift. Since we are already inside the cluster the following OC CLI picks up the current openshift instance.

Reply:
Thank you for explaining